test/00_transform/FFT.cu

```diff
@@ -92,6 +92,50 @@ TYPED_TEST(FFTTestComplexTypes, FFT1D1024C2C)
   MATX_EXIT_HANDLER();
 }
 
+TYPED_TEST(FFTTestComplexTypes, FFT1DFWD1024C2C)
+{
+  MATX_ENTER_HANDLER();
+  const index_t fft_dim = 1024;
+  this->pb->template InitAndRunTVGenerator<TypeParam>(
+      "00_transforms", "fft_operators", "fft_1d_fwd", {fft_dim, fft_dim});
+
+  tensor_t<TypeParam, 1> av{{fft_dim}};
+  tensor_t<TypeParam, 1> avo{{fft_dim}};
+  this->pb->NumpyToTensorView(av, "a_in");
+
+  // example-begin fft-1-fwd
+  // Perform a 1D FFT from input av into output avo with FORWARD scaling (1/N). Input and output sizes will be deduced by the
+  // type of the tensors and output size.
+  fft(avo, av, fft_dim, 0, FORWARD);
+  // example-end fft-1
+  cudaStreamSynchronize(0);
+
+  MATX_TEST_ASSERT_COMPARE(this->pb, avo, "a_out", this->thresh);
+  MATX_EXIT_HANDLER();
+}
+
+TYPED_TEST(FFTTestComplexTypes, FFT1DORTHO1024C2C)
+{
+  MATX_ENTER_HANDLER();
+  const index_t fft_dim = 1024;
+  this->pb->template InitAndRunTVGenerator<TypeParam>(
+      "00_transforms", "fft_operators", "fft_1d_ortho", {fft_dim, fft_dim});
+
+  tensor_t<TypeParam, 1> av{{fft_dim}};
+  tensor_t<TypeParam, 1> avo{{fft_dim}};
+  this->pb->NumpyToTensorView(av, "a_in");
+
+  // example-begin fft-1
```

Comment:
I think these should go in increasing order instead of being called fft-1. We already have up to fft-5:

https://github.com/NVIDIA/MatX/blob/main/docs_input/api/dft/fft/fft.rst

so these would be 6/7